alerta-covid/spec/requests/api/v1/areas_spec.rb

```diff
@@ -0,0 +1,29 @@
+ require 'rails_helper'
+
+RSpec.describe "/areas", type: :request do
+  let(:valid_attributes) {
+    {name: "demo"}
+  }
+
+  let(:invalid_attributes) {
+    {name: ""}
+  }
+
+  before(:each) do
+    FactoryBot.create(:area, id:1, institution_id: 1 ,name: "area")
+  end
+
+  describe "GET /Api/V1/index" do
+    it "renders a successful response" do
+      get "Api/V1/areas"
```

Comment:
Did you tried with `api/v1/areas`? 
I've never seen using capitals in routes.

I think the router does support it, but let's try to use conventional paths

Reply:
the mentioned routes were modified in the next commit [test spec part2](https://github.com/BC-NOV-20-Proyectos/alerta-covid-web-team01/pull/13/commits/6a68bcb443badd8caea6a43616319dae0962bfc8)